### ISSUE TITLE
Excuse me ,Can we have the VirtualSMC build?

### ISSUE THREAD

**Issue by whoisme007** [corpnewt/Lilu-and-Friends#18]
https://github.com/acidanthera/VirtualSMC

very like the Lilu and friends,very useful ,now acidanthera make the VirtualSMC,can Lilu and Friends makebuild it?

**Comment by corpnewt** [comment]
It will be added eventually, yes. I'm holding back a bit so that acidanthera isn't bombarded with issues and questions unrelated to issues with the kext itself for the moment as it's in its first iteration, and a sort of "dev preview".

**Comment by whoisme007** [comment]
thank you,but i have a question. VirtualSMC is a EFI file which needs to put into the Driver64UEFI folder,and When I xcodebuild it ,the answer is a efi file named VirtualSMC.efi.
but the lilu-and-friends build it ,and comes out a VirtualSMC.kext?are this two things same?

**Comment by corpnewt** [comment]
Thanks for bringing that to my attention - I've added another option to build all the targets in the VirtualSMC repo and zip them up - this contains everything.

**Comment by whoisme007** [comment]
Thank you so much.After I use the LIlu-and friends,put the VirtualSMC.efi into the DRiver64UEFI,put the SmcLightSensor.kext and SmcProcessor into the kexts,and reboot,My i5 7500 temp down to 34degree from 37degree.What a good day!The lilu-and friends is the most useful lilu build toolbox!
and I don't know if this is my personal problem? if i don't put the VIrtualSMC.kext into kexts,everything is fine.
If i put it into ,when i power on the PC,it stops at the IGPU,no matter I set the GFX0toIGPU or not ,no matter I set the -disablegfxfirmware or not .No matter I set the Inject Intel Graphics or not.No matter I set the Injects kexts yes or no .
My computer is i5 7500,gtx1030,Ga-H110m, iMac18.2,video output use the HDMI2.0,base on GPU.
